.kokoro/tests/get_changed_files.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+import requests
+
+
+_PR_URL_TEMPLATE = (
+    'https://api.github.com/repos/googleapis/google-cloud-python/pulls/'
```

Comment:
I think we  eventually started to see GitHub quota issues (too many requests with no associated token). If we start to see that we can add a read-only GH token for this request to use.

Reply:
Thanks! Maybe I can do that now